src/Notebook/notebook-help.tsx

```diff
@@ -122,6 +122,15 @@ function AnonymizedResultsHelp() {
           Click here for details.
         </DocsLink>
       </Paragraph>
+      <Paragraph>
+        The{' '}
+        <DocsLink page="operation" section="suppress-bin">
+          {' '}
+          suppress bin
+        </DocsLink>{' '}
+        denoted by <Text code>*</Text> column values, is shown first. It indicates the total anonymized count of all
```

Comment:
In docs we say combined, here we say total (my mistske from the suggedtions). Maybe worth making consistent so that we don't confuse users?